docs/book/src/developer/providers/v1.3-to-v1.4.md

```diff
@@ -24,6 +24,7 @@ maintainers of providers and consumers of our Go API.
 - `MachineHealthCheckSuccededCondition` condition type has been removed.
 - `CloneTemplate` and `CloneTemplateInput` has been removed.
 - The option `--list-images` from `init` subcommand has been removed.
+- `ClusterTopologyManagedFieldsAnnotation` field has been removed.
```

Comment:
Similar here - and for the other items on this list as in https://github.com/kubernetes-sigs/cluster-api/pull/7844

Could we add the package names to the doc to make it easier to understand (can do it in a follow up if you prefer)